src/current/v23.1/monitoring-and-alerting.md

```diff
@@ -189,6 +189,685 @@ If you rely on external tools for storing and visualizing your cluster's time-se
 
 When storage of time-series metrics is disabled, the DB Console Metrics dashboards in the DB Console are still available, but their visualizations are blank. This is because the dashboards rely on data that is no longer available.
 
+### Critical nodes endpoint
+
+The critical nodes status endpoint is used to:
+
+- Check if any of your nodes are critical.  A node is 'critical' if that node becoming unreachable would cause [replicas to become unavailable](ui-cluster-overview-page.html#replication-status).
+- Check if any ranges are [under-replicated or unavailable](ui-cluster-overview-page.html#replication-status). This is useful when determining whether a node is ready for [decommissioning](node-shutdown.html#decommissioning).
+- Check if any of your cluster's data placement constraints (set via [Multiregion SQL](multiregion-overview.html) or direct [configuration of replication zones](configure-replication-zones.html)) are being violated. This is useful when implementing [data domiciling](data-domiciling.html).
+
+{{site.data.alerts.callout_info}}
+This status endpoint supersedes the deprecated [Replication Reports](query-replication-reports.html) SQL API.
```

Comment:
Nit: Is "SQL API" necessary here? I always get confused about when we actually refer to an official "API". Based on the link, it looks like this just refers to some system tables?

Also, what does it technically mean that the status endpoint supersedes the Replication Reports? Is the returned info more accurate, or is it more that we recommend the former over the latter?

EDIT: I see from the next article that it's the former, due to architectural changes, so it's probably worth stating that here as well.

Reply:
everything is an API!!!1!  SQL is just another thing to program against.  I'm not sure if it's necessary to call it that but I wanted to contrast it to this thing, which is an HTTP API.  I updated this a little to say this is an "HTTP" thing vs that which is a "SQL" thing

updated to say

> This HTTP status endpoint supersedes the deprecated [Replication Reports](query-replication-reports.html) SQL API. Due to architectural changes in CockroachDB, the SQL queries described on that page will not result in correct output.